src/app/view/web/WebChessView.ts

```diff
@@ -3,20 +3,51 @@ import { ChessBoardView } from '../ChessBoardView';
 import { ViewEventBus } from '../events/ViewEventBus';
 import { ViewEvent } from '../events/ViewEvent';
 import { SquareWasClicked } from '../events/SquareWasClicked';
+import { PiecesBoardPositions } from '../../model/Types';
 
 export class WebChessView implements ChessBoardView {
   constructor(private readonly viewEventBus: ViewEventBus, private readonly parent: HTMLElement = document.body) {}
 
-  showChessBoard(): void {
+  showChessBoard(piecesPositions: PiecesBoardPositions): void {
     const chessboard: Chessboard = new Chessboard('chessBoardId', 'chessboard', (position) =>
       this.viewEventBus.publish(new SquareWasClicked(position)),
     );
     this.parent.appendChild(chessboard.createBoard());
+
+    this.renderPiecesOnBoard(piecesPositions);
   }
 
   listenOn<EventType extends ViewEvent = ViewEvent>(eventType: EventType['eventType'], reaction: (event: EventType) => void): void {
     this.viewEventBus.listenOn(eventType, reaction);
   }
 
   showSelectedPiece(position: { x: number; y: number }): void {}
+
+  private renderPiecesOnBoard(piecesPositions: PiecesBoardPositions) {
+    let piecesSquareId: string;
+    let pieceImage: string;
+    let pieceName: string;
+    let pieceSide: string;
+
+    for (const piece in piecesPositions) {
```

Comment:
Im mniej modyfikacji / forow tym lepiej :) Jak dla mnie to zrobisz w sposob mnie jwiecej:
```
piecesPositions.map(...).forEach()
```
No bo mapujemy najpierw na div z svg, potem chcemy wszystkie dodac do widoku.